package.json

```diff
@@ -8,18 +8,18 @@
     "templateVersion": "7.2.4"
   },
   "scripts": {
+    "kbn": "node ../../kibana/scripts/kbn",
     "lint": "eslint **/*.js",
     "start": "plugin-helpers start",
     "test:server": "plugin-helpers test:server",
     "test:browser": "plugin-helpers test:browser",
     "build": "plugin-helpers build",
-    "postinstall": "plugin-helpers postinstall",
     "gather-info": "node gather-info.js"
   },
   "devDependencies": {
-    "@elastic/eslint-config-kibana": "^0.14.0",
```

Comment:
As we discussed, keep this as it was before, but describe the linking approach in readme.md